node/events/psv2Webhook.ts

```diff
@@ -29,6 +29,7 @@ export async function psv2Webhook(ctx: EventContext<Clients>) {
     profileId: event.profileId,
     operation: event.operation,
     subject: event.entity,
+    author: event.author,
```

Comment:
Why we are duplicating this field instead of just adding it inside meta to the missing operation?

Reply:
Because the meta is inside the document and it's not possible to get the document for the delete operation.